flocker/node/_config.py

```diff
@@ -1245,35 +1248,50 @@ def deployment_from_configuration(deployment_configuration, all_applications):
     return set(nodes)
 
 
-def model_from_configuration(applications, deployment_configuration):
+def parse_desired_configuration(application_configuration,
```

Comment:
`Deployment.from_deployment_and_application_config`?  Now I'm trying to figure out how a `Deployment` differs from a hypothetical `Cluster`.  Probably it doesn't, I guess.  So maybe this is `Deployment.from_deployment_and_application_config` (or just `from_config`?) and the above API is some other private helper.